modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -26,17 +26,29 @@ def fetch_aud
       Settings.oidc.isolated_audience.claims
     end
 
+    protected
+
+    def source_name
+      if v0?
+        request.headers['X-Consumer-Username']
+      else
+        user = header_request? ? @current_user : target_veteran
+        "#{user.first_name} #{user.last_name}"
+      end
+    end
+
```

Comment:
Is this temporary branching logic? Should we handle versioning concerns in a different namespace - i.e. having parallel implementations rather than branching logic inside the base controller?

Reply:
source_name is used within this base controller for a simple find that is apparently shared between v0 and v1
https://github.com/department-of-veterans-affairs/vets-api/blob/aab8083f1af918d91359aae8474a893d0e2358c1/modules/claims_api/app/controllers/claims_api/application_controller.rb#L43